### ISSUE TITLE
[bug]  ar4100 slippery stairs don't work properly

### ISSUE THREAD

**Issue by lynxlynxlynx** [gemrb/gemrb#509]
>  ar4100: The two sets of slippery stairs don't work properly (jumptopoint is skipped) - (41pstail.baf)
>   *  while it also uses Entered (true only once), recheck when isoverme is changed to handle LastTrigger

Doesn't block #499. Switching it off works fine, plus there's always the abishai way of entry. Most likely a duplicate of the reason for #271.
